### ISSUE TITLE
Update Platform Tooling database clusters to use smaller instances in the dev environment

### ISSUE THREAD

**Issue by plittlewood-rpt** [reapit/foundations#10919]
**Background context**
There are two RDS clusters using 2 medium RDS instances each in the dev environment. From a cost saving perspective we should alter the instance sizing in the dev environment to use smaller instances

**Comment by bashleigh** [comment]
Just updated these instances to be aurora 3.03.1 and the minimum size you can have is t3 medium which I believe they are? Will keep this issue open for refernce and just in case I'm incorrect.

**Comment by plittlewood-rpt** [comment]
Yes this is correct. t4g.medium is marginally cheaper but we'll see more benefit if we don't run multi AZ clusters in the dev environment
